.github/workflows/servicenow.py

```diff
@@ -226,20 +234,24 @@ def get_cmr_id_operation():
     "closeCode": "Successful",
     "notes": "The change request is closed as the change was released successfully"
   }
-  response = requests.post(servicenow_cmr_url, headers=headers, json=data)
-  jsonParse = json.loads(response.text)
+  response = requests.post(SERVICENOW_CMR_URL, headers=headers, json=data)
+  json_parse = json.loads(response.text)
 
   if response.status_code != 200:
-    print("POST failed with response code: ", response.status_code)
+    print(f"{POST_FAILURE_MESSAGE} {response.status_code}")
     print(response.text)
     sys.exit(1)
-  elif find_string_in_json(jsonParse, "error"):
-    print("CMR closure failed with response code: ", response.status_code)
+  elif find_string_in_json(json_parse, "error"):
+    print(f"CMR closure failed with response code: {response.status_code}")
     print(response.text)
     sys.exit(1)
   else:
-    print("CMR closure was successful: ", response.status_code)
+    print(f"CMR closure was successful: {response.status_code}")
     print(response.text)
 
-  print("Change Management Request in ServiceNow was successful.")
-  print ("You can find the change record in ServiceNow https://adobe.service-now.com/now/change-launchpad/homepage, by searching for this ID: ", cmr_id)
+  print("Change Management Request has been sent to the queue and is being processed.")
+  print(f"You can find the change record in ServiceNow https://adobe.service-now.com/now/change-launchpad/homepage, by searching for this ID: {cmr_id}")
```

Comment:
Should we also send a slack message both for when this succeeds and fails?

Reply:
Yeah, I'll add that in.